### ISSUE TITLE
document._currentScript not needed

### ISSUE THREAD

**Issue by warpech** [Juicy/imported-template#28]
This custom element has a variant of the following line:

```js
var script = document._currentScript || document.currentScript;
var template = script.previousElementSibling;
```

`document._currentScript` was only needed in IE11, but Starcounter does not support that browser anymore. 

Therefore, this code can be simplified to:

```js
var template = document.currentScript.previousElementSibling;
```

If we want to drop support for IE11.

**Comment by warpech** [comment]
This is blocked on https://github.com/Starcounter/Guidelines/issues/32

**Comment by rianby64** [comment]
I'd like to suggest you another point of view around this idea of HTML-importing... there's a good summary [here](https://github.com/m3co/pseudoimport-html/issues/64)

**Comment by warpech** [comment]
This is really off-topic in this issue, so I responded there. Thanks for finding our work :)

**Comment by warpech** [comment]
Get back to this after #30

**Comment by warpech** [comment]
As checked in HTML Imports polyfill that is part of WC.js V1, `document._currentScript` is not needed. I will add a PR to remove that.

**Comment by rianby64** [comment]
My apologizes that I wrote here off-topic. Good luck with your projects, @warpech .

**Comment by warpech** [comment]
Don't be, we are doing what we can to be pragmatic about `imported-template`. Thanks for being so active in this field as well.

We are committed long-term to use a standards way to load chunks of HTML. As the spec develops, we will most probably migrate towards HTML Modules.

**Comment by kegor** [comment]
Closed because it was merged in #45.
